homeassistant/components/isy994/switch.py

```diff
@@ -158,6 +158,7 @@ def __init__(
 
     async def async_added_to_hass(self) -> None:
         """Subscribe to the node control change events."""
+        await super().async_added_to_hass()
```

Comment:
This is not a control node so it shouldn't call async_added_to_hass or it will get the wrong events here